### ISSUE TITLE
Addition of mytsid

### ISSUE THREAD

**Issue by xpiREC** [par0noid/ts3admin.class#65]
Hi, I noticed that the `mytsid` element is missing, in documentation, I will try to release pulls.

**Comment by ReconV** [comment]
https://forum.teamspeak.com/threads/137798-Release-TeamSpeak-3-Server-3-5-0
https://github.com/par0noid/ts3admin.class/pull/66

**Comment by lgund** [comment]
That´s correct. I´ve merged the missing parameter. But isn´t also missing the public ban method to ban via mytsid?

> What you can do is to use the ClientQuery and the command banadd mytsid=...... to add such ban manually into the ban list.
